### PR TITLE
Add authoring organizations to programs in search results

### PR DESCRIPTION
**Pull request by rlucioni**
@clintonb how off-base am I here?

### CODE REVIEW

File: course_discovery/apps/course_metadata/search_indexes.py [hunk 1]
```diff
@@ -117,6 +123,7 @@ class ProgramIndex(BaseIndex, indexes.Indexable, OrganizationsMixin):
     organizations = indexes.MultiValueField(faceted=True)
     authoring_organizations = indexes.MultiValueField(faceted=True)
     credit_backing_organizations = indexes.MultiValueField(faceted=True)
+    organization_bodies = indexes.CharField()
```
Comment (rlucioni) [inline course_discovery/apps/course_metadata/search_indexes.py]: Should this be an unfaceted `MultiValueField`? It's a list of serialized organizations.

Comment (clintonb) [inline course_discovery/apps/course_metadata/search_indexes.py]: Yes.

**Comment by clintonb** [comment]
@rlucioni this looks good to me.

**Comment by rlucioni** [comment]
@clintonb cool, thanks for taking a look. I'll look into adding some tests later this evening.

**Comment by rlucioni** [comment]
@clintonb please review. @AlasdairSwan FYI.

File: pylintrc [hunk 2]
```diff
@@ -80,7 +80,7 @@ disable =
 	too-many-arguments,
 	too-many-locals,
 	unused-wildcard-import,
-	duplicate-code,invalid-name,missing-docstring
+	duplicate-code,invalid-name,missing-docstring,cyclic-import
```
Comment (clintonb) [inline pylintrc:83]: We don't want to disable this globally. Disable it for the one line that matters. There may be future cases where a circular import is not intended.

**Comment by clintonb** [comment]
👍 (once pylint comment addressed)

File: pylintrc (hunk 2 above)
Comment (rlucioni) [inline pylintrc:83]: Unfortunately, that doesn't appear to currently be possible. See https://github.com/PyCQA/pylint/issues/850. Pylint won't accept a module-scoped disable directive for this.

Comment (clintonb) [inline pylintrc:83]: Please add a note, and link to that issue in this file.

Comment (rlucioni) [inline pylintrc:83]: I've done that in `pylintrc_tweaks`. I was under the impression we should try to avoid editing this file.

Comment (clintonb) [inline pylintrc:83]: I missed that. Carry on.

**Pull request merged**